tests/php/WildcardDomainFieldTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
```

Comment:
`tests/php/SubsitesVirtualPageTest.yml` will need to have namespacing added to it